{{cookiecutter.github_project_name}}/tests/karma.conf.js

```diff
@@ -27,6 +27,14 @@ module.exports = function (config) {
 
```

Comment:
And line 12 of this file, we need to include:

```
    files: [
      'node_modules/babel-polyfill/dist/polyfill.js',
      { pattern: "tests/src/**/*.ts" },
      { pattern: "src/**/*.ts" },
    ],
```

Reply:
Whoops! Thanks for double checking. I just added babel-polyfill and confirmed locally that the tests pass and `jupyter lab` properly shows the sample widget.